rest/src/main/java/io/syndesis/rest/v1/handler/setup/OAuthAppHandler.java

```diff
@@ -128,7 +150,7 @@ private String getPropertyTaggedAs(Connector connector, String name) {
         return null;
     }
 
-    private void setPropertyTaggedAs(Connector connector, Map<String, String> configuredProperties, String name, String value) {
+    private static void setPropertyTaggedAs(Connector connector, Map<String, String> configuredProperties, String name, String value) {
         TreeSet EMPTY = new TreeSet();
```

Comment:
Just noticed this `EMPTY` from the previous PR, perhaps use a constant from `java.util.Collections` instead (can be refined in a new PR).